lemur/tests/test_integration_db_and_models.py

```diff
@@ -0,0 +1,239 @@
+# This file consists of testing for the creation of objects and their
+# relationship in db
+import sys
+sys.path.append('../..')
```

Comment:
I want to separate testing scripts from the files to be tested so I have to change the path in order to access the files to be tested.